projects/ad9213_evb/vcu118/README.md

```diff
@@ -0,0 +1,32 @@
+# EVAL-AD9213/VCU118 HDL Project
+
+## Building the project
+
+The parameters configurable through the `make` command, can be found below, as well as in the **common/ad9213_evb_bd.tcl** file; it contains the default configuration.
+
+```
+cd projects/ad9213_evb/vcu118
+make
+```
+
+All of the RX link modes can be found in the [AD9213 data sheet](https://www.analog.com/media/en/technical-documentation/data-sheets/AD9213.pdf). We offer support for only a few of them.
+
+If other configurations are desired, then the parameters from the HDL project (see below) need to be changed, as well as the Linux/no-OS project configurations.
+
+The overwritable parameters from the environment:
```

Comment:
This project is not parameterizable, it has this fixed configuration, which it should be specified here, and also remove the text where you mentioned the common/ad9213_evb_bd.tcl file